int32 bot_id
bool cleaning
float32 battery_level
string status
